mipengine/node/tasks/tables.py

```diff
@@ -49,8 +50,8 @@ def create_table(
 
     Returns
     ------
-    str
-        The name of the created table in lower case
+    str(TableInfo)
```

Comment:
should also change the return type in the function declaration

Reply:
It's still a string, since it's serialized.

Reply:
What we do currently is:
```
    Returns
    ------
    str(TableInfo)
      A TableInfo object in a jsonified format
```

which I guess is pretty close to what you said. Isn't it?